src/leiningen/monolith.clj

```diff
@@ -201,9 +201,11 @@
     (let [config (config/read!)
           subprojects (get-subprojects project config)
           start-from (some-> (:start opts) ffirst read-string)
-          targets (-> (dependency-map subprojects)
-                      (cond->
-                        (:subtree opts) (u/subtree-from (project-sym project)))
+          relevant-subprojects (-> (dependency-map subprojects)
```

Comment:
You don't really need  the extra level of threading here anymore, so you could just make the `cond->` form top-level.